src/tools/genpolicy/rules.rego

```diff
@@ -809,7 +816,9 @@ allow_storages(p_storages, i_storages, bundle_id, sandbox_id) {
     i_count := count(i_storages)
     print("allow_storages: p_count =", p_count, "i_count =", i_count)
 
-    p_count == i_count
+    # If it's not a virtio-blk storage, genpolicy creates a superfluous
```

Comment:
Let's discuss examples - I don't understand why we would end up with superfluous storage objects.

Reply:
For managed-csi, there's no storage object created by the shim. For cc-managed-csi, there is one, representing the virtio-blk device.

Reply:
Hopefully, the persistentVolumeClaim you used in the change below knows that the user wants cc-managed-csi (and not managed-csi), and then we add the missing block Storage to the policy? Then the new Storage wouldn't be superfluous anymore and the storage would continue to work.